vuepress/docs/next/docs/curate/bundle-presentation-config.md

```diff
@@ -1,10 +1,11 @@
-# How to customize bundle info shown in Entando App Builder
+# Customize Bundle Info Shown in App Builder
+
+An example of how a bundle is displayed in the Local Hub of the App Builder is shown below:
 
 ![Hub user interface in Entando App Builder](./img/local-hub-page.png)
 
-In the image above you can see an example of a bundle as is displayed in the Hub user interface inside the Entando App Builder.
 
-The corresponding custom-resource file on Kubernetes is similar to this:
+The corresponding custom resource file on Kubernetes contains content similar to the following:
```

Comment:
this is minor but should this be "on Kubernetes" or "in kubernetes"? Does a CR file live/act in or on Kubernetes, this always kind of sits odd with me in different contexts

Reply:
i think it's "on"... kubernetes runs things on a cluster of hosts... i also think the concept of layering applies, where things are on layers, not in them. pretty sure "in kubernetes" refers to conceptual architecture and "on kubernetes" refers to implementation